wrappers/winrt/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ add_definitions (-DUNICODE -D_UNICODE)
 
 if (MSVC)
     add_definitions (
-        -DNOMINMAX
+        -DNOMINMAX -wd4996
```

Comment:
That is hopefully not there to simply silence the deprecation warnings that I recently introduced? ;)

Reply:
Yes it is. It's necessary since the project is configured to flag warnings as build errors.

Reply:
Well, I'd suggest to rather change that setting then to allow for builds with warnings. But this win-rt stuff is definitively not my area of expertise :).

Reply:
It's ok if we got few warnings to remind us things to be done. But the real issue with leaving warnings is that it's easy get ignored and get accumulated with the time. And when it becomes too many, it becomes serious problem but no one wants to fix ;)